app/services/ClickHandler.coffee

```diff
@@ -72,7 +74,8 @@ class ClickHandler
     false
 
   unselectAll: ->
-    @RunningSum.display ''
+    unless $('#running-sum').html() is @RunningSum.solutionOperatorString
```

Comment:
Should we wrap calls to jQuery? Seems like having a method like
```coffee
@runningSumEl: () ->
   $('#running-sum')
```
or
```coffee
@runningSumEl = $('#running-sum')
```

would allow for more ability to change the id.